misc/scripts/install-local.sh

```diff
@@ -311,7 +316,7 @@ if [[ -n $patch ]]; then
 fi
 
 export pkgdir="/usr/src/pacstall/$name"
-prepare
+fakeroot -- prepare
```

Comment:
From the man page I think it's trying to run a "command" called `prepare` not a function.

See this syntax: ` fakeroot [-l|--lib library] [--faked faked-binary] [--] [command]`

Reply:
Actually I tested with a mini script before putting it into pacstall. And it worked, but it doesn't in pacstall. That's what I'm trying to figure out

Reply:
You have to run `export -f loca`